### PR TITLE
cmd: encapsulate les relative cli options

### PR DESCRIPTION
**Pull request by rjl493456442**
This PR is based on #19814.

This PR encapsulates all les relative options and categories these options into `LES`.

### CODE REVIEW

File: cmd/geth/main.go [hunk 1]
```diff
@@ -93,10 +93,12 @@ var (
...
-		utils.LightServFlag,
-		utils.LightBandwidthInFlag,
-		utils.LightBandwidthOutFlag,
-		utils.LightPeersFlag,
+		utils.LightServeFlag,
+		utils.LightLegacyServFlag,
+		utils.LightIngressFlag,
+		utils.LightEgressFlag,
+		utils.LightMaxPeersFlag, // Deprecated in favor of light.maxpeers, remove in 2021
+		utils.LightLegacyPeersFlag,
```
Comment (rjl493456442) [inline cmd/geth/main.go]: Should the `// Deprecated in favor of light.maxpeers` added here?

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -949,8 +960,26 @@ func setIPC(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
-// setUltraLight configures the ultra light client settings from the command line flags.
-func setUltraLight(ctx *cli.Context, cfg *eth.Config) {
+// setLes configures the les server and ultra light client settings from the command line flags.
+func setLes(ctx *cli.Context, cfg *eth.Config) {
+	if ctx.GlobalIsSet(LightLegacyServFlag.Name) { // Deprecated in favor of light.maxpeers, remove in 2021
```
Comment (rjl493456442) [inline cmd/utils/flags.go]: `// Deprecated in favor of light.serve, remove in 2021`

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -949,8 +960,26 @@ func setIPC(ctx *cli.Context, cfg *node.Config) {
...
+		cfg.LightServ = ctx.GlobalInt(LightLegacyServFlag.Name)
+	}
+	if ctx.GlobalIsSet(LightServeFlag.Name) { // Deprecated in favor of light.maxpeers, remove in 2021
```
Comment (rjl493456442) [inline cmd/utils/flags.go]: rm `// Deprecated in favor of ...`

Comment (karalabe) [reply cmd/utils/flags.go]: whoops

File: cmd/utils/flags.go (hunk 2 above)
Comment (karalabe) [reply cmd/utils/flags.go]: Good catch, removed altogether.

File: cmd/utils/flags.go (hunk 3 above)
Comment (karalabe) [reply cmd/utils/flags.go]: Good catch, removed altogether.

File: cmd/geth/main.go (hunk 1 above)
Comment (karalabe) [reply cmd/geth/main.go]: Removed all the other instances, code started to look weird with all these deprecation repetitions.

**Pull request merged**